\RequirePackage{etex}
\documentclass[AMA,STIX1COL]{WileyNJD-v2}

\articletype{Research Article}%

\received{26 April 2016}
\revised{6 June 2016}
\accepted{6 June 2016}

\raggedbottom

\begin{document}

\title{Extending statistical models for source attribution of zoonotic diseases: A study of campylobacteriosis}

\author[1]{Sih-Jing Liao*}

\author[1]{Jonathan Marshall}

\author[1]{Martin L. Hazelton}

\author[2]{Nigel P. French}

\authormark{Sih-Jing Liao \textsc{et al}}


\address[1]{\orgdiv{Institute of Fundamental Sciences},\orgname{Massey University}, \orgaddress{\state{Palmerston North}, \country{New Zealand}}}

\address[2]{\orgdiv{mEpiLab, Hopkirk Research Institute}, \orgname{Massey University}, \orgaddress{\state{Palmerston North}, \country{New Zealand}}}

\address[3]{\orgdiv{Org Division}, \orgname{Org Name}, \orgaddress{\state{State name}, \country{Country name}}}

\corres{*Sih-Jing Liao, Institute of Fundamental Sciences, Massey University, Palmerston North, New Zealand. \email{S.J.Liao@massey.ac.nz}}

%\presentaddress{This is sample for present address text this is sample for present address text}

\abstract[Summary]{Preventing and controlling zoonoses through the design and implementation of public health policies requires a thorough understanding of transmission pathways. Modeling jointly the epidemiological data and genetic information of microbial isolates derived from cases provides a methodology for tracing back the source of infection. In this paper, the attribution probability for human cases of campylobacteriosis for each source, conditional on the extent to which each case resides in a rural compared to urban environment, is estimated. A model that incorporates genetic data and evolutionary processes is applied alongside a newly developed genetic-free model. We show that inference from each model is comparable except for rare microbial genotypes, and that the effect of ``rurality'' may be modeled linearly on the logit scale, with increasing rurality leading to increasing likelihood of ruminant-sourced campylobacteriosis.}

\keywords{source attribution; \textit{Campylobacter}; genetic model; Dirichlet prior; HPD interval; DIC}

\jnlcitation{\cname{%
\author{S.J. Liao}, 
\author{J. Marshall}, 
\author{M.L. Hazelton}, and 
\author{N.P. French}
\author{T. Woollings}} (\cyear{2018}), 
\ctitle{Extending statistical models for source attribution of zoonotic diseases: A study of campylobacteriosis}, \cjournal{Statistics In Medicine}.} %, \cvol{2018;00:1--6}.}

\maketitle

%\footnotetext{\textbf{Abbreviations:} ANA, anti-nuclear antibodies; APC, antigen-presenting cells; IRF, interferon regulatory factor}

\section{Introduction}
Modeling of disease surveillance data to explore patterns of infectious diseases has had a long history in public health. Infectious diseases can cause high economic and medical costs due to morbidity and mortality. In recent decades, the annual number of global deaths caused by infections has leveled off at approximate 15 million and may remain at this level for the next three decades \cite{DyeC, WHOM}. In order for such an enormous health burden to be reduced, preventing and controlling infectious diseases becomes extraordinarily important, and our ability to intervene depends on how much we know about the nature of disease transmission. For zoonotic diseases, where transmission to humans from animal reservoirs may be via food, water, through environmental contamination or direct contact with animals, knowledge of the potential sources and pathways of infection is key to reducing the burden of disease. Quantifying the burden of illness attributable to each pathway and source provides valuable information to health professionals making decisions on public health policy for disease control.

Zoonotic transmission from animals to humans may be complex, involving many sources and exposures linked by differing pathways. For instance, infected wild birds may contaminate environmental water and cause disease spread to water users, either humans or other animals \cite{Wagen}. Tracing the source of infection is thus crucial to increasing the ability to implement risk management and intervention \cite{Wilso, Morel}. In terms of statistical analysis, modeling zoonoses requires an advanced approach with the focus changed from just epidemiology to a combination of epidemiology, evolutionary genetics and biology \cite{Muell}. Hence, some source attribution models have been proposed to estimate the number of cases attributable to different sources by utilizing epidemiological information and the association with genotypes found in humans and sources \cite{vanP, Hald, MullA}.

\begin{table}
  \begin{center}
	\begin{tabular}{rrrrrrrr}
	\toprule
	ST & \textit{asp}A & \textit{gln}A & \textit{glt}A & \textit{gly}A & \textit{pgm} & \textit{tkt} & \textit{unc}A\\
	\midrule
	2026 & 10 & 1 & 16 & 19 & 10 & 5 & 7\\
	474 & 2 & 4 & 1 & 2 & 2 & 1 & 5\\
	2343 & 2 & 4 & 5 & 2 & 10 & 1 & 5\\
	403 & 10 & 27 & 16 & 19 & 10 & 5 & 7\\
	\bottomrule
	\end{tabular}
  \end{center}
  \caption{The allelic profiles of a selection of genotypes, composed of seven allele numbers at each of the seven loci.}
  \label{tab:mlst}
\end{table}

The genetic information used in such integrated models is typically derived from molecular genotyping that groups closely related organisms together \cite{Cotta}. Typically molecular typing methods, such as multilocus sequence typing (MLST) \cite{Dingl, Coll, Urwin} might be used, which utilizes nucleotide sequences of internal fragments of a small set of housekeeping genes. Such sequences have sufficient variation to distinguish differing pathogen lineages, while being relatively stable within lineages. Each unique nucleotide sequence (allele) at each housekeeping gene (locus) is assigned a number, and the set of numbers across all loci (the allelic profile) is then taken as the genotype, which is assigned a sequence type (ST) number.

For the pathogen \textit{Campylobacter} which causes campylobacteriosis, a worldwide gastrointestinal disease in humans, the commonly used 7-gene MLST scheme consists of housekeeping genes \textit{asp}A (aspartase A), \textit{gln}A (glutamine synthetase), \textit{glt}A (citrate synthase), \textit{gly}A (serine hydroxymethyltransferase), \textit{pgm} (phosphoglucomutase), \textit{tkt} (transketolase), and \textit{unc}A (ATP synthase $\alpha$ subunit). An illustrative example of MLST data for \emph{Campylobacter} is presented in Table~\ref{tab:mlst}. It shows that the genotypes ST-2026 and ST-474 have different allelic combinations across all seven loci, while ST-403 differs from ST-2026 only at \textit{gln}A, and ST-2343 differs from ST-474 at \textit{glt}A and \textit{pgm}. The different allelic profiles enable comparison of gene similarities or dissimilarities so that an association between sources and infected cases can be made, by comparing the distribution of genotypes from human cases with those from potential reservoirs.

The first step for attribution models that utilize genetic information is building the sampling distribution of genotypes among each putative source. This may range from using the proportion of each observed genotype \cite{Hald} on each source through to utilizing allelic profile information to derive mutation and recombination rates within each source, and migration rates between each source \cite{Wilso}. A key question is whether more complex genetic models yield superior attribution results or whether a significantly simpler model may suffice, but few authors in the literature have addressed this point. This becomes more important as model complexity extends to include epidemiological covariates. We are therefore motivated to develop a simple model in order to assess the additional information that the more complex models provide.

In this paper we develop statistical models for source attribution, and demonstrate their use on a case study of human campylobacteriosis conducted in New Zealand \cite{Marsh}. Human campylobacteriosis is caused mainly by \textit{C. jejuni} and \textit{C. coli} which are the dominant species associated with approximately 80\% and 15\% of illnesses, respectively \cite{Guert}. Common symptoms of infection are diarrhea, abdominal pain and fever, however, a severe complication named Guillain Barr\'{e} syndrome may develop, which is a life-threatening disease that weakens the nervous system and leads to paralysis in limbs and respiration \cite{Hanha}. The pathogen can be spread between animals, or from animals and wild birds to humans. Transmission routes may be via drinking contaminated water, eating undercooked animal food products, or handling animal food products that are already contaminated by faeces. In this study, we compare the performance of the asymmetric Island model \cite{Wilso}, which considers genetic evolution when estimating the genotype sampling distribution on each source, to a simple model that utilizes only the prevalence of each type to derive the sampling distribution. We then extend both models in a Bayesian context to incorporate covariates, exploring the effect of human case rurality on attribution results via a linear trend on the logit scale or with separate categories, and performing model comparison.

\section{Models for source attribution with and without microbial genetic information}
\subsection{Notation and assumptions}
The goal of attribution models is to estimate the probability that the observed human cases arise from each putative source. Our data comprise microbial genotype information from each observed human case obtained from analysis of stool samples, and also from a pool of non-human cases corresponding to potential zoonotic sources of disease.

Given this genotyping information, we first estimate the sampling distribution of genotypes for each source, and then estimate the appropriate combinations of those genotype distributions that most likely give rise to the set of genotypes observed among human cases. Specifying first the sampling distribution of genotypes found on sources is fundamental for the purpose of not only exploring how it affects the source attribution probability, but also investigating the difference in attribution effect made between different genetic models.

Suppose we have isolates collected from human and non-human cases, of which $H$ isolates belong to humans, and the remaining $N$ isolates are categorized in $J$ groups as the major sources attributed to the infection. Let $I$ genotypes be the total number of unique types detected from all isolates, and denote $n_j$ as the marginal frequency of types found in source $j$, where $\sum_j^J n_j = N$. Typically, the number of detected types $I$ is smaller than the sample size of isolates as multiple isolates will be of the same type.

The likelihood of observing human cases with types $\text{ST}_1, \text{ST}_2, \ldots \text{ST}_H$ follows a multinomial distribution, which via the law of total probability may be expressed as
\begin{align}
  L\Big(\text{ST}_1, \text{ST}_2, \cdots, \text{ST}_H \Big)=\prod_{h=1}^{H}\sum_{j=1}^{J} p\Big(\text{ST}_h \vert \text{source } j\Big) p\Big(\text{source }j\Big),
  \label{Olikh}
\end{align}
where $p(\text{ST}_h \vert \text{source }j)$ is the probability that genotype $\text{ST}_h$ arises from the sampling distribution of source $j$, and $p(\text{source }j)$ is the attribution probability that a random human case is infected from source $j$. Given we know $p(\text{ST}_h \vert \text{source }j)$, estimation of $p(\text{source }j)$ may be found by optimising the likelihood \eqref{Olikh}, for example using a Metropolis Hastings algorithm within a Bayesian context, with suitable priors on $p(\text{source }j)$.

The asymmetric Island model \cite{Wilso} adopted in the source attribution study for human campylobacteriosis \cite{Marsh} utilizes the allelic profile information for each genotype in an evolutionary model, estimating mutation and recombination probabilities within, and migration probabilities between, each source `island'. It thus estimates $p(\text{ST}_h \vert \text{source }j)$ indirectly, by first estimating the evolutionary parameters, and then deriving the sampling distributions. This allows the asymmetric Island model to estimate the likelihood of observing a genotype on a source when it has not been previously observed.

To discover the effect of incorporating genetic information at the allelic profile level as used in the asymmetric Island model, a simple model is developed for the genotype sampling distribution. With the assumption that the observed distribution of genotypes is representative of the true distribution, we model observed genotypes using a multinomial distribution. Let $x_{ij}$ denote the count of genotype ST-$i$ found in source $j$ with the probability $\pi_{ij}$, where $i=1, \ldots, I$ and $j=1, \ldots, J$. To make inference about $\pi_{ij}$, the likelihood is of a multinomial form,
\begin{align}
        L(\boldsymbol{\pi}_{j} ; \boldsymbol{x}_{j}) & = \frac{n_j!}{\prod_{i=1}^{I} x_{ij}!} \prod_{i=1}^I \pi_{ij}^{x_{ij}}, \notag
\end{align}
where $x_{ij}$ can be $0$, indicating genotypes are yet observed on the sources; $n_j=\sum_{i=1}^I x_{ij}$ is the total count of all types found on source $j$ and $\pi_{ij}$ is subject to $\sum_{i=1}^I \pi_{ij} =1$, and $0 \leq \pi_{ij} \leq 1$. As the family of Dirichlet distributions is a conjugate pair for the multinomial distribution, assume the prior for $\boldsymbol{\pi}_{j}$ follows a Dirichlet density with parameters $\boldsymbol{\alpha}_{j}$,
\begin{align}
  p(\boldsymbol{\pi}_{j}) & \propto \prod_{i=1}^{I}\pi_{ij}^{\alpha_{ij}-1}. \notag
\end{align}
Then the posterior for $\boldsymbol{\pi}_{j}$ takes the form of a Dirichlet probability function with parameters $(\boldsymbol{\alpha}_{j}+\boldsymbol{x}_{j}-\boldsymbol{1})$,
\begin{align}
        p(\boldsymbol{\pi}_{j} \vert \boldsymbol{x}_{j} ) & \propto L(\boldsymbol{\pi}_{j} ; \boldsymbol{x}_{j}) p(\boldsymbol{\pi}_{j}) \notag \\
       & \propto \prod_{i=1}^{I}\pi_{ij}^{\alpha_{ij} + x_{ij}-1}, \quad \alpha_{ij} >0. \notag
 \end{align}
To express the belief that every isolate is equally likely \emph{apriori}, the parameter of the Dirichlet prior is assumed as $\boldsymbol{\alpha}_{j}=\boldsymbol{1}$. Therefore, $p(\text{ST}_h \vert \text{source }j)$ can be obtained by simulating from the Dirichlet posterior.

Lastly, to estimate the attribution probability, 100 samples of $p(\text{ST}_h \vert \text{source }j)$ are generated using the asymmetric Island or simple models. For each posterior sample, we infer $p(\text{source }j)$ using the likelihood~\eqref{Olikh}. This has the effect of integrating over the uncertainty in $p(\text{ST}_h \vert \text{source }j)$ when estimating $p(\text{source }j)$.

\subsection{Campylobacteriosis data}

The source attribution study described here uses human case data ($H=1,460$) collected in the Manawatu region of New Zealand from March 2005 to December 2014. In addition, over the same time period and from the same geographical location, isolates $N=2,128$ sampled from chicken carcasses, cattle, sheep, environmental water, wild birds, and so on, were also taken, and categorised into four groups  representing major sources of infection: poultry, ruminants, water and others. The total number of unique genotypes from all isolates is $I=348$, with 36\% of genotypes found among human cases. Table~\ref{tab:counts} lists five common genotypes found in human and source isolates, the first four of which are frequently observed in human cases. As found in other studies, ST-45 and ST-474 are detected mainly in poultry, while ST-42 and ST-2026 are detected mainly in ruminants \cite{Muell, Coll, Cart}. The fifth genotype, ST-2381 is not found among human cases, appearing only in the water and other sources, in this case being found in Pukeko and Takah\={e} birds from the Rallidae family \cite{french2014evolution}.

The data also contains location information, in the form of an ordinal classification of urban and rural areas with seven levels coded from -3 to 3: highly rural/remote area, rural area with low urban influence, rural area with moderate urban influence, rural area with high urban influence, independent urban area, satellite urban area and main urban area. Approximately 8\% of individuals in the Manawatu dataset contain no information about the location, which we assume are missing at random. Table~\ref{tab:rurality} lists the remainder of typed human cases in each classification of rurality as well as the population from the 2006 and 2013 Census \cite{Stat1, Stat2}. The case rate per 100,000 population is illustrated in Figure~\ref{fig:case_rates}, where we see the burden of infection in urban areas drop remarkably from 2008, coinciding with an intervention in the poultry industry implemented by the New Zealand Food Safety Authority (NZFSA) in 2007 and 2008. It shows the intervention improved infection rates in urban areas, however, it only has a temporary effect in rural areas.

\begin{table}
  \begin{center}
	\begin{tabular}{rrrrrr}
	\toprule
	ST & Human & Poultry & Ruminants & Water & Others\\
	\midrule
	42 & 59 & 7 & 53 & 10 & 2\\
	45 & 149 & 155 & 10 & 21 & 54\\
	474 & 247 & 60 & 15 & 5 & 9\\
	2026 & 28 & 0 & 40 & 5 & 2\\
	2381 & 0 & 0 & 0 & 60 & 3\\
	\bottomrule
	\end{tabular}
  \end{center}
  \caption{The frequency of five genotypes found from human and four source isolates.}
  \label{tab:counts}
\end{table}

\begin{table}
%  \begin{threeparttable}
  \centering
	\begin{tabular}{rlrrr}
	\toprule
	Rurality scale & Description & Human cases & 2006 & 2013\\
	\midrule
	-3 & Highly rural/remote area & 16 & 1,572 & 1,527\\
	-2 & Rural area with low urban influence & 103 & 8,382 & 8,316\\
	-1 & Rural area with moderate urban influence & 124 & 10,392 & 10,734\\
	0 & Rural area with high urban influence & 78 & 6,579 & 7,155\\
	1 & Independent urban area & 240 & 28,611 & 28,188\\
	2 & Satellite urban area & 187 & 19,725 & 20,526\\
	3 & Main urban area & 596 & 76,047 & 78,108\\
	\bottomrule
	\end{tabular}
      \caption{The number of human cases in each rurality class from 2005--2014, and the population size in 2006 and 2013, in the Manawatu region of New Zealand.}
      \label{tab:rurality}
\end{table}


\begin{figure}
  \centering
  \includegraphics[width=.8\linewidth]{figures/fig_case_rates.pdf}
  \caption{Case rates per 100,000 population in urban and rural areas of the Manawatu region of New Zealand from 2005 through 2014. An intervention in the poultry industry conducted in 2007 and 2008 resulted in a decreasing incidence of campylobacteriosis in the following years, particularly in urban areas.}
  \label{fig:case_rates}
\end{figure}

\section{Model fitting for source attribution on rurality scale}

\subsection{Incorporating covariates}

Previously we described how to estimate the marginal probabilities that a randomly selected human case is due to a given source. To extend this analysis so as to include individual level covariates, we need to calculate subject-specific attribution (conditional) probabilities. To that end, let $F_{jh}$ denote the attribution probability of source $j$ for the $h^\textrm{th}$ human case, with constraints $\sum_{j=1}^J F_{jh} =1$ and $0 \leq F_{jh} \leq 1$, where $j=1, \ldots, J$. We model the probabilities $F_{jh}$ using a linear model on the logit scale, that is,
\begin{align}
  F_{jh}  = \frac{\exp (f_{jh})}{\sum_{j=1}^4 \exp(f_{jh})},
  \label{capF}
\end{align}
where $f_{4h}=0$ is treated as the baseline of $f_{jh}$. Consider the case where the genotype data for each human case is supplemented by $p$ additional variables. A general model of $f_{jh}$ with linear combinations of the variables, $c_1, \ldots, c_p$, then has the form
\begin{align}
f_{jh}  = \alpha_{j} + \beta_{j1} c_{1h} + \beta_{j2} c_{2h} + \ldots + \beta_{jp} c_{ph},  \notag
\end{align}
for the $h$-th individual. Note that if there is a single categorical variable with $L$ levels, then $F_{jh}$ and $f_{jh}$ will take no more than $L$ distinct values. In a slight abuse of notation, we will at times refer to $F_{jh}$ in which the $h$ index refers to the factor level, rather than to a particular subject at that level.

To apply the general model of $f_{jh}$ to the campylobacteriosis data, assume $z$ is the variable ranging from -3 to 3 representing the classified location of each human case. Then two ways of treating the variable $z$ in model fitting are proposed: one is to treat it as numeric, and the other as categorical. To differentiate the performance between the two fitted models, we link ``the linear model'' and ``the categorical model'' to the first and the latter fitted model, respectively. Hence, the linear prediction function for source $j$ for each human case given the degree of rurality is a numeric variable and can be written as
\begin{align}
  f_{jh} = \alpha_{j} + \beta_{j} z_{h},
  \label{modL}
\end{align}
where $z_{h}$ can be any number of the seven scales if case $h$ was from such a degree of rurality. Conversely, if we treat each of the seven rurality degrees as an indicator with a superscript number $d$, which corresponds with the position of the category ranged from -3 to 3, the model~\eqref{modL} can be rewritten as
\begin{equation}
  f_{jh} = \beta_{1j} z_{1h}+ \beta_{2j} z_{2h} + \ldots + \beta_{dj} z_{dh} + \ldots +  \beta_{7j} z_{7h},
  \label{modC}
\end{equation}
where
\begin{equation*}
z_{dh} =
\left\{\begin{array}{rcl}
1 && \mbox{if case $h$ is in the category $d$,} \notag \\ 0 && \mbox{otherwise}.
\end{array}\right.
\end{equation*}
As a consequence, the estimated attribution probabilities are obtained via equation~\eqref{capF} after fitting the data to model~\eqref{modL} or to model~\eqref{modC}.

\subsection{MCMC algorithm}
In the interest of quantifying the uncertainty of the posterior attribution probability, we perform Bayesian inference for source attribution probabilities using Markov chain Monte Carlo (MCMC) methods. Assume the priors on parameters of interest in model~\eqref{modL} and model~\eqref{modC} follow a standard normal distribution. Let $\boldsymbol{\theta}$ denote the vector of parameters, with elements $\theta_{t}$ for $t=1, \ldots, T$. For example, $\boldsymbol{\theta}=\{\alpha_1, \alpha_2, \alpha_3, \beta_1, \beta_2, \beta_3\}$ in model~\eqref{modL} or $\boldsymbol{\theta}=\{\beta_{11}, \beta_{12}, \ldots, \beta_{dj}, \ldots, \beta_{73}\}$ in model~\eqref{modC}. We use the Metropolis-Hastings algorithm to update $\theta_{(t)}$ and hence $f_{jh}$ and $ F_{jh}$, in a Markov chain with a length of 11,000 iterations. The first 1,000 samples are removed as the burn-in period (during which time the chain converges) and the sequence is thinned every 100$^{th}$ sample to reduce computer storage. Here are the steps in detail:

\begin{enumerate}
\setcounter{enumi}{-1}
\item Sample $T$ random values from $N(0,1)$ as initial values of the parameter set $\boldsymbol{\theta}$ for model~\eqref{modL} or model~\eqref{modC}.
\item Sample a permutation $P_T$ of $\{1, \ldots, T\}$.\label{algorithm_loop}
\item For each $t \in P_T$:
\begin{enumerate}
\item Propose a candidate $\boldsymbol{\theta}^*$ with $\theta_{(t)}$ updated by a normal proposal distribution, $Q(\theta_{(t)}^*, \theta_{(t)}) = N(\theta_{(t)}, 1)$.

\item Use $\boldsymbol{\theta}^*$ to calculate a new set of $f^*$ for source $j$, $j=1, 2, 3$, via model~\eqref{modL} or model~\eqref{modC}, and find the associated $F^*$ for each case by putting the vector $(f^*, f_4=0)$ in equation~\eqref{capF}.

\item Compute the acceptance probability $\alpha$, a product of the likelihood ratio, the proposal ratio and the prior ratio, given by
\begin{align*}
\alpha=\frac{L\Big(F^*; \text{ST}\Big)}{L\Big(F; \text{ST}\Big)}\frac{Q\Big(\theta_{(t)} \vert \theta_{(t)}^*\Big)}{Q\Big(\theta_{(t)}^* \vert \theta_{(t)}\Big)}\frac{p\Big(\theta_{(t)}^*\Big)}{p\Big(\theta_{(t)}\Big)},
\end{align*}
where the likelihood $L\Big(F^*; \text{ST}\Big)$ is given by equation \eqref{Olikh}. \item Sample $u$ from $U(0,1)$. If $u < \min \left\{1, \alpha \right\}$, the proposals $f^{(*)}$, $F^{(*)}$ and $L\Big(F^{(*)}; \text{ST}\Big)$ are utilised for the next iteration; otherwise, the previous values are retained.
\end{enumerate}
\item Repeat from step \ref{algorithm_loop} for the given number of iterations.

\end{enumerate}

All code for fitting the genotype distributions, estimating attribution probabilities and producing figures, along with the data are available in the R package `islandR` from \url{https://github.com/jmarshallnz/islandR}.

\section{Results}

\begin{figure}
\centering
\includegraphics[width=.9\linewidth]{figures/fig_attribution.pdf}
\caption{Posterior mean attribution ($F$) of human cases with 80\% credible intervals for source: poultry (red), ruminants (blue), water (green) and others (purple) over the rurality scales from highly rural area to main urban areas. The attribution is generated from both the linear and the categorical models, given the sampling distribution of genotypes with evolutionary information (the asymmetric Island model) or without any genetic information (the Dirichlet model).}
\label{fig:attribution}
\end{figure}

\begin{table}
  \begin{center}
    \begin{tabular}{crrrrr}
      \toprule
      & \multicolumn{2}{c}{Island model}  & \multicolumn{1}{c}{} & \multicolumn{2}{c}{Dirichlet model} \\
      Data & Linear & Categorical & & Linear & Categorical\\ \midrule
      whole period  & 7913.6 & 7943.3  & & 10328.4  & 10389.8 \\
      post-intervention period  & 5413.4  & 5439.5  & & 6615.1   & 6668.9 \\
      \bottomrule
    \end{tabular}
  \end{center}
  \caption{DIC values for the linear model and for the categorical model applied to the data from 2005 to 2014 (whole period) and to the data from 2008 to 2014 (post-intervention period) given the genotypic information is from the asymmetric Island model or the Dirichlet model.}
  \label{tab:dic}
\end{table}

\begin{figure}
\centering
\includegraphics[width=.9\linewidth]{figures/fig_genotype_dist.pdf}
\caption{Posterior probability for each source for four sequence types from the asymmetric Island (red) and Dirichlet (blue) models, assuming that each source is \emph{apriori} equally likely.}
\label{fig:genotype_dist}
\end{figure}

\begin{figure}
\centering
\includegraphics[width=.9\linewidth]{figures/fig_intervention.pdf}
\caption{Posterior mean attribution ($F$) of human cases from 2005--2007 and 2008--2014 with 80\% credible intervals for poultry (red), ruminants (blue), water (green), and other sources (purple) over the rurality scales from highly rural area to main urban area (Table~\ref{tab:rurality}). The attribution is generated using the linear model given the sampling distribution of genotypes with evolutionary information (the asymmetric Island model) or without any genetic information (the Dirichlet model).}
\label{fig:intervention}
\end{figure}

Posterior attribution of human cases of campylobacteriosis ($F$) with 80\% credible intervals for each source are illustrated for each rurality grade in Figure~\ref{fig:attribution}. The graphs are categorized by the type of genetic model (asymmetric Island or Dirichlet) and the manner in which the rurality variable is modelled (categorical or linear on the logit scale).

Overall, the attribution results are relatively stable irrespective of the genetic model or how rurality is represented in the attribution model. The majority of human cases are attributed to ruminants and poultry, with more cases attributed to ruminants in rural areas and more cases attributed to poultry in urban areas. For the Dirichlet and asymmetric Island models, the linear and categorical models of rurality show broadly the same trend, suggesting that the additional flexibility given by the categorical model is not required, and that the shift in attribution as rurality changes is adequately modelled by a linear trend on the logit scale. The linear model has the advantage of tighter credible intervals as it can share data across the 7 levels of rurality, resulting in a clearer separation of ruminant and poultry attribution, particularly in highly rural areas where the data are sparse.

There are some small differences between the genotype models, with the Dirichlet model showing a greater attribution to poultry (ranging from 40\% in highly rural areas to 75\% in main urban centres) than the asymmetric Island model (ranging from 30\% in rural areas to 65\% in urban centres). This occurs in both the linear and categorical models. Interestingly, the asymmetric Island model attributes approximately 7\% of human cases across all rurality levels to sources other than poultry, ruminants and water, and gives a small attribution to water in highly rural areas, whilst the Dirichlet model indicates that both these sources are unimportant.

We use deviance information criterion (DIC) for model comparison. DIC values obtained from our MCMC runs are displayed in Table~\ref{tab:dic}. Overall there is a clear signal that a linear representation of rurality (on the logit scale) is adequate. Note that the asymmetric Island and Dirichlet models are not directly comparable by DIC as the likelihoods are on different scales: the Dirichlet model assumes all potential sequence types have been observed so that $\sum_j p(\boldsymbol{\pi}_j) = 1$, whereas the asymmetric Island model allows for unobserved sequence types so that $\sum_j p(\boldsymbol{\pi}_j) < 1$.

The small differences in attribution observed between the asymmetric Island and Dirichlet models may be due to the additional genetic information available to the asymmetric Island model. To illustrate this, Figure~\ref{fig:genotype_dist} shows the probability of source given genotype for a selection of four genotypes, assuming \emph{apriori} that each source was equally likely. Genotypes ST-403 and ST-2343 are observed primarily in humans ($n=6$ cases each) with ST-403 not being observed among the sources, and ST-2343 being observed once in poultry, so that the Dirichlet model has little information available to distinguish between sources. The asymmetric Island model, however, can exploit the genetic relationship between genotypes. ST-403 differs at just one locus from ST-2026, a type observed frequently in human cases and ruminant isolates, while ST-2343 differs at two loci from common genotype ST-474, observed frequently in human cases and poultry isolates (Tables~\ref{tab:mlst} and \ref{tab:counts}). Thus, the asymmetric Island model can clearly assign ST-403 to ruminants and ST-2343 to poultry, whilst the Dirichlet model cannot distinguish between sources. In contrast, both models provide similar probabilities for ST-2026 and ST-474 which are both observed frequently.

As noted previously, a major public health initiative in 2007 led to a significant reduction in the number of cases of campylobacteriosis in New Zealand. In order to examine the effects of this change on attribution probabilities, we repeated the analysis by including an interaction with time period (2005--2007 and 2008--2014). Figure~\ref{fig:intervention} shows that the general trend in attribution by rurality for each of the time periods utilising a linear trend on the logit scale to incorporate rurality. There is a clear difference, with a significantly lower attribution to poultry (and correspondingly higher attribution to ruminants) in all but the most rural of areas, being strongest in highly urban areas. Thus, although the intervention did not eliminate infection arising from poultry \cite{MuelM}, the reduction highlights the significant improvement in contribution of poultry to the disease, particularly in urban areas where most cases lived.

\section{Discussion}
Models that determine the source of human infection, particularly for zoonotic pathogens that originate in animal populations, are of considerable value to public health policy makers. However, such models may be complex, particularly when utilising evolutionary models. An outstanding question is whether such complexity is required, or whether a simpler model may work as effectively. Here we developed a relatively simple model to estimate the attribution probability for each source of \textit{Campylobacter} infection. This model differs to the asymmetric Island model, in that it does not model pathogenic evolution, opting instead to use infer the sampling distribution of genotypes directly from the observed count data.

Our results showed that the Dirichlet model and the asymmetric Island model showed largely similar results for final attribution, with both models demonstrating a clear effect of rurality on attribution: cases in rural areas are more likely to have originated from ruminants, while those in main urban centers are more likely to be of poultry origin. As most people in the Manawatu region live in urban centers, this highlights the importance of poultry as a reservoir for campylobacteriosis, which is well established in the literature \cite{MullA, Marsh, MullM, Leve}.

When we ran models allowing attribution probabilities to differ before and after the intervention in the poultry industry in 2007, we saw a clear difference, with much lower poultry (and higher ruminant) attribution, particularly in main urban centers in 2008--2014. Considering that most people in the Manawatu region live in urban areas, and that case rates in urban areas decreased from 2008 onwards, it is clear that this intervention coincided with a dramatic reduction in poultry attributed illness as reported elsewhere \cite{AnnS}. Given that campylobacteriosis cases in rural areas are mostly attributed to ruminant sources, and that case rates in these areas have been higher than those in urban centers since 2008, there is a clear need for public health interventions to focus in this area.

While the overall attribution was consistent between the Dirichlet and asymmetric Island models, it would be expected that the conditional probabilities for a given genotype might differ markedly. For those genotypes observed infrequently (or not at all) among the sources the Dirichlet model has little information while the asymmeteric Island model can exploit information from cases with similar (but not identical) genetic profiles. In the case of MLST data with just 7 loci, the majority of human cases and source isolates come from a relatively small number of sequence types which are observed often. Thus the Dirichlet model performs well, as it has sufficient observations to estimate the genotype distribution well where the bulk of the data lie. It is only those genotypes that are rarely observed where it performs poorly, but as they are rarely observed, they don't contribute significantly to the overall attribution. In other circumstances, such as where we have many more than 7 loci, we would expect to have many more rare genotypes, so that the Dirichlet model might provide little useful information. At the extreme example of whole genome MLST (wgMLST) where each isolate would typically be unique, it would provide essentially no information at all. In such circumstances, however, the asymmetric Island model would be expected to still perform well, assuming that information could still be transferred between similar genotypes.

As with any Bayesian analysis, it is of interest to examine the sensitivity of the results to the choice of prior distributions. We originally used standard normal priors for regression coefficients on the logit scale. We also considered priors with $\sigma^2=4$, and while this meant that $f$ tended to drift further from 0, the resulting attribution probabilities $F$ did not change, largely as the attribution is dominated by poultry and ruminant sources, with the water source in particular being close to zero. Thus, $f_\mathsf{poultry}$ and $f_\mathsf{ruminants}$ are positive, while $f_\mathsf{water}$ is negative and the magnitude of these can increase without making all that much difference to their corresponding $F$'s. The prior on $f$ thus tends to restrict this ill-behaviour rather than acting as a strong constraint on attribution probabilities. The prior $\boldsymbol{\alpha}_j$ in the Dirichlet also makes little difference if kept small, as it most strongly effects genotypes that are rare, which do not contribute significantly to the overall attribution. The prior can be thought of as data augmentation such that $\boldsymbol{\alpha}_j=\boldsymbol{1}$ is equivalent to adding a single observation of each genotype to source $j$. Thus, large values of $\boldsymbol{\alpha}_j$ will cause the genotype distributions across sources to look more similar, and hence result in equal attribution to each source.

Future research might focus on possible extensions to these models. One direction is to adapt the models with additional covariates, which might include age, occupation, and other risk factors such as contact with animals. For example, there is evidence that children in rural areas are at higher risk of campylobacteriosis through contact with farm animals \cite{MullM, Spencer2012}. Another direction is in expanding the role of water. In these models, we've assumed that water is a source of human campylobacteriosis infection, but water differs from the other food and environmental sources in that it is not an amplifying reservoir for \emph{Campylobacter} \cite{Wagen}. In contrast, genotypes found in water might be expected to originate in the other sources present here, particularly ruminants and wild birds, but also potentially from humans as well via discharge of unprocessed human waste. Hence, water acts as a transmission pathway from sources to humans, being both an end point (reduced water quality from faecal contamination) and a source (human consumption of water, either recreationally or through untreated water supplies). While there is presently little evidence that water is an important source for human campylobacteriosis from the current models, these are fit on sporadic cases of campylobacteriosis. However, water is known as a key source of outbreaks of campylobacteriosis, such as the large outbreak in Havelock North, New Zealand in 2016 where an estimated 5,500 out of 14,000 residents became ill \cite{dia_campy_report1}. Thus, characterising the source of \emph{Campylobacter} found in water has important implications for both water quality and public health.

\section*{Acknowledgements}

We thank Mid Central Public Health Services, and the Molecular Epidemiology and Public Health Laboratory (mEpiLab, Massey University) for data collection, and the Ministry of Primary Industries for funding the Manawatu campylobacteriosis sentinel surveillance site.

\bibliography{BibArt}

\end{document}
